themes/classic/modules/ps_bestsellers/views/templates/hook/ps_bestsellers.tpl

```diff
@@ -26,9 +26,9 @@
   <h2 class="h2 products-section-title text-uppercase">
     {l s='Best Sellers' d='Shop.Theme.Catalog'}
   </h2>
-  <div class="products">
-    {foreach from=$products item="product"}
-      {include file="catalog/_partials/miniatures/product.tpl" product=$product}
+  <div class="products" itemscope itemtype="http://schema.org/ItemList">
```

Comment:
You'll hate me but can we add this block in a file and include where it's needed.
It will be easier to change `itemtype="http://schema.org/ItemList` for example if it's needed
```twig
  <div class="products" itemscope itemtype="http://schema.org/ItemList">
    {foreach from=$products item="product" key="position"}
      {include file="catalog/_partials/miniatures/product.tpl" product=$product position=$position}
    {/foreach}
  </div>
```

Reply:
@PierreRambaud No problem, I understand it :)

Not sure where I put this file : themes/classic/templates/catalog/_partials/productlist.tpl ?

Reply:
seems good!

Reply:
Done